packages/flame_lottie/lib/src/lottie_renderer.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flame/effects.dart';
+import 'package:flutter/rendering.dart';
+import 'package:lottie/lottie.dart';
+
+class LottieRenderer extends CustomPainter {
```

Comment:
Any reason for making this a `CustomPainter`?

This is not intentioned to be used in a `CustomPaint` widget right?

Reply:
i just would like to adhere to the [CustomPainter](https://api.flutter.dev/flutter/rendering/CustomPainter-class.html) signature, nothing more than that.

No, the renderer is not intended be used in a CustomPaint Flutter widget. Theoretically you could use it in such a widget yes, but its not directly intended for it.

From looking at Flutter docs nothing says that this interface is only used for renderers that are in use with CustomPaint widgets.

If you prefer I can remove it, I don't mind and have no strong opinion about it

Reply:
Yeah, I feel this can cause confusion, since this is a flame based library I think we should make it explict so.

My suggestion here would be to follow a little bit how `flame_rive` is implemented: https://github.com/flame-engine/flame/blob/main/packages/flame_rive/lib/src/rive_component.dart#L78